src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -530,6 +530,10 @@ Please change the value to an assembly-qualifed type name which inherits from '{
     <comment>The following are literal names and should not be translated: Java.Interop.DoNotPackageAttribute
 {0} - The assembly name</comment>
   </data>
+  <data name="APT2264" xml:space="preserve">
+    <value>This is probably caused by the project exceeding the Max Path length. See https://learn.microsoft.com/en-us/xamarin/android/errors-and-warnings/apt2264 for details.</value>
```

Comment:
I think we are supposed to remove the `en-us` part from MS docs links. Also it seems like they call this `Maximum Path Length Limitation`, maybe we can stick with that name?

```suggestion
    <value>This is probably caused by the project exceeding the Windows maximum path length limitation. See https://learn.microsoft.com/xamarin/android/errors-and-warnings/apt2264 for details.</value>
```

Is this page not there yet? Going to be a new page?